pyproject.toml

```diff
@@ -32,25 +32,24 @@ keywords=["Scade", "Wrapper", "Integration"]
 
 [project.optional-dependencies]
 build = [
-    "build==1.2.1",
-    "twine==5.1.1"
+    "build==1.2.2.post1",
+    "twine==6.1.0"
 ]
 tests = [
     # 8.0.2 at most: https://github.com/microsoft/PTVS/issues/7853
-    "pytest==8.0.2",
-    "pytest-cov==5.0.0"
+    "pytest==8.4.1",
+    "pytest-cov==6.2.1"
 ]
 doc = [
-    "ansys-sphinx-theme[autoapi]==1.2.4",
+    "ansys-sphinx-theme[autoapi]==1.5.3",
     "autodoc_pydantic==2.2.0",
-    "jupyter_sphinx==0.5.3",
-    "nbsphinx==0.9.5",
-    "numpydoc==1.8.0",
+    "numpydoc==1.9.0",
     "sphinx==8.1.3",
     "sphinx-autodoc-typehints==2.3.0",
-    "sphinxcontrib-globalsubs==0.1.1",
+    "sphinxcontrib-globalsubs==0.1.2",
     "sphinxcontrib-httpdomain==1.8.1",
     "sphinx-copybutton==0.5.2",
+    "sphinx-design===0.6.1",
     "sphinx-jinja==2.0.2",
```

Comment:
`Sphinx` design and sphinx jinja are already included in  "ansys-sphinx-theme[autoapi]", dependency.
```suggestion
```